test-common/src/main/scala/org/scalajs/testcommon/RPCCore.scala

```diff
@@ -133,13 +160,13 @@ private[scalajs] abstract class RPCCore {
   }
 
   private def makeReply[T: Serializer](id: Long, result: Try[T]): String = {
-    result.map(makeMsg(ReplyOK, id, _)) match {
+    result.map(makeRpcMsg(ReplyOK, id, _)) match {
       case Success(m) => m
-      case Failure(t) => makeMsg(ReplyErr, id, t)
+      case Failure(t) => makeRpcMsg(ReplyErr, id, t)
     }
   }
 
-  private def makeMsg[T: Serializer](opCode: OpCode, id: Long,
+  private def makeRpcMsg[T: Serializer](opCode: OpCode, id: Long,
```

Comment:
We usually keep acronyms all-uppercase in camelCase identifiers, so this should be named `makeRPCMsg`.